runtime/vm/VMSnapshotImpl.cpp

```diff
@@ -615,6 +615,7 @@ VMSnapshotImpl::removeUnpersistedClassLoaders()
 	for (UDATA i = 0; i < count; i++) {
 		J9ClassLoader *currentClassLoader = removeLoaders[i];
 
+#if 0
```

Comment:
Please explain why this code is not simply removed.

Reply:
@lzhou2025 You need to respond to comments explicitly. Simply pushing code to the PR does not indicate that the feedback has been addressed.